index.html

```diff
@@ -36,5 +36,151 @@ <h5 class="connect">LET'S CONNECT</h5>
+              <li><img src="./img/Counter.png" alt="counter"></li>
+              <li class="tonic-detail2">2015</li>
+            </ul>
+            <p class="project-detail">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div>
+                <ul class="buttons">
+                    <li><button class="int-button">html</button></li>
+                    <li><button class="int-button">css</button></li>
+                    <li><button class="int-button">JavaScript</button></li>
+                </ul>
+            </div>
+            <div><button class="see">See Project</button>
+            </div>
+          </div>
+
+          <div class="water3">
+            <div class="project3"><img src="./img/Snapshoot-project3.png" alt="project3"></div>
+            <h2 class="tonic">Tonic</h2>
+            <ul class="info">
+              <li class="tonic-detail">CANOPY</li>
+              <li><img src="./img/Counter.png" alt="punto"></li>
+              <li class="tonic-detail2">Back End Dev</li>
+              <li><img src="./img/Counter.png" alt="counter"></li>
+              <li class="tonic-detail2">2015</li>
+            </ul>
+            <p class="project-detail">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div>
+                <ul class="buttons">
+                    <li><button class="int-button">html</button></li>
+                    <li><button class="int-button">css</button></li>
+                    <li><button class="int-button">JavaScript</button></li>
+                </ul>
+            </div>
+            <div><button class="see">See Project</button>
+            </div>
+
+          </div>
+
+          <div class="water4">
+            <div class="project4"><img src="./img/Snapshoot-project4.png" alt="project4"></div>
+            <h2 class="multi-post">Multi-Post Stories</h2>
+            <ul class="info">
+              <li class="tonic-detail">CANOPY</li>
+              <li><img src="./img/Counter.png" alt="punto"></li>
+              <li class="tonic-detail2">Back End Dev</li>
+              <li><img src="./img/Counter.png" alt="counter"></li>
+              <li class="tonic-detail2">2015</li>
+            </ul>
+            <p class="project-detail">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div>
+                <ul class="buttons">
+                    <li><button class="int-button">html</button></li>
+                    <li><button class="int-button">css</button></li>
+                    <li><button class="int-button">JavaScript</button></li>
+                </ul>
+            </div>
+            <div><button class="see">See Project</button></div>
+          </div>
+        </div>
+      </section>
+    </div>
+
+    <section>
+      <div class="garden2">
+        <div class="water5">
+          <h1 class="abaout-me">About <br>Myself</h1>
```

Comment:
- [ ] Please try to follow the order of headings in your document for accessibility and SEO reasons. so you need to have one  `h1` tag and at the top of the page in the document and other headings can be used multiple times.